Sources/Extensions/UIApplication+KeyWindow.swift

```diff
@@ -54,13 +58,20 @@ public extension UIViewController {
         }
         
         // presentedViewController
-        if let presentedViewController = viewController.presentedViewController {
+        if let presentedViewController = viewController.presentedViewController,
+            (includeSystemViewControllers || !presentedViewController.isSystemViewController) {
             return visibleViewController(presentedViewController)
         }
         
         return viewController
     }
     
+    /// Is `self` a presented system `UIViewController`
+    /// - TODO: Add greater support
```

Comment:
Agree with this `TODO` :D We should probably add things in here such as `MFMessageComposeVC`